.travis.yml

```diff
@@ -136,7 +136,7 @@ jobs:
             php: 7.3
             name: Test Coverage
             script:
-                - phpdbg -qrr -d memory_limit=-1 vendor/bin/phpunit --coverage-clover coverage.xml
+                - pecl install pcov; composer require pcov/clobber; vendor/bin/pcov clobber; vendor/bin/phpunit --configuration phpunit.xml --coverage-clover coverage.xml 
```

Comment:
should be in standalone lines instead of `x; y;`

Reply:
Also reference to the post would be great, as I've never heard of it and it might be difficult to debug it in a year

Reply:
Will wait for the actual perf number and adjust in case its faster

Reply:
Coverage is down to ~1min30

Reply:
Nice, go for it then :+1:

Reply:
Better remove fatal errors though: https://travis-ci.org/rectorphp/rector/jobs/633259038#L547